docs/how-tos/how_to_skip_rmas_in_returns.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 2
+---
+
+# How to skip RMAs in returns
```

Comment:
I like how specific this how-to is, because skipping RMAs is definitely the most common use case for customizing the eligibility validators. However, I'm wondering if this might prevent users with other use cases from finding this how-to. What do you think?

Reply:
Thanks for pointing that out. It also bugged me, but I didn't know how to fix it.

I've changed the title to "How to customize the default eligibility in returns: skipping RMAs". What do you think? Being such a common problem, I think it makes sense to keep it in the title for better visibility.

Reply:
How about this one?
```How to customize return eligibility rules: skipping RMAs```

Reply:
I like it. Updated.